tests/test_browser.py

```diff
@@ -2767,10 +2767,15 @@ def in_html(expected, args=[]):
         in_html('200')
 
   @requires_graphics_hardware
-  def test_glfw3(self):
+  @parameterized({
+    'no_gl': (['-DCLIENT_API=GLFW_NO_API'],),
+    'gl_es': (['-DCLIENT_API=GLFW_OPENGL_ES_API'],)
+  })
+  def test_glfw3(self, args):
+    print(args)
```

Comment:
```suggestion
```
No need for this - there's enough in the test name. (If we have this in other tests we should probably remove it there too...)

Reply:
Removed it. There are quite a few tests using this pattern.